tests/component-plugin/src/main/resources/META-INF/mule-artifact/mule-plugin.json

```diff
@@ -7,7 +7,9 @@
       "exportedPackages": [
         "org.mule.functional.api.component",
         "org.mule.functional.api.exception",
-        "org.mule.functional.api.notification"
+        "org.mule.functional.api.notification",
+        "org.mule.functional.config",
```

Comment:
I don't remember TBH... I will need to re-run the full test suite in order to tell you